lib/providers/providers.dart

```diff
@@ -94,59 +90,57 @@ final providers = [
-    create: (_) => AuthBloc(),
-    update: (_, service, authNotifier, settingsNotifier, authBloc) => authBloc!
-      ..update(service, authNotifier,
-          settingsNotifier), // AuthNotifier seems broken, shouldn't need to be updated so often
-    // dispose:
-  ),
-  ProxyProvider<SettingsNotifier, BackupService>(
-    create: (_) => BackupService(),
-    update: (_, settings, backupService) => backupService!..update(settings),
-  ),
+  // ChangeNotifierProxyProvider<HttpService, BalanceNotifier>(
+  //   create: (context) => BalanceNotifier(),
+  //   update: (context, http, balance) => balance!..update(http: http),
+  // ),
+  // ChangeNotifierProxyProvider<HttpService, DhoNotifier>(
+  //   create: (context) => DhoNotifier(),
+  //   update: (context, http, notifier) => notifier!..update(http: http),
+  // ),
+  // ChangeNotifierProxyProvider<HttpService, RateNotifier>(
+  //   create: (context) => RateNotifier(),
+  //   update: (context, http, rate) => rate!..update(http: http),
+  // ),
+  // ChangeNotifierProxyProvider<HttpService, VotedNotifier>(
+  //   create: (context) => VotedNotifier(),
+  //   update: (context, http, vote) => vote!..update(http: http),
+  // ),
+  // ChangeNotifierProxyProvider<HttpService, VoiceNotifier>(
+  //   create: (context) => VoiceNotifier(),
+  //   update: (context, http, balance) => balance!..update(http: http),
+  // ),
+  // ChangeNotifierProxyProvider<HttpService, PlantedNotifier>(
+  //   create: (context) => PlantedNotifier(),
+  //   update: (context, http, balance) => balance!..update(http: http),
+  // ),
+  // ChangeNotifierProxyProvider<HttpService, ProfileNotifier>(
+  //   create: (context) => ProfileNotifier(),
+  //   update: (context, http, members) => members!..update(http: http),
+  // ),
+  // Provider(
+  //   create: (_) => BiometricsService(LocalAuthentication()),
+  // ),
+  // ProxyProvider3<BiometricsService, AuthNotifier, SettingsNotifier, AuthBloc>(
+  //   create: (_) => AuthBloc(),
+  //   update: (_, service, authNotifier, settingsNotifier, authBloc) => authBloc!
+  //     ..update(service, authNotifier,
+  //         settingsNotifier), // AuthNotifier seems broken, shouldn't need to be updated so often
+  //   // dispose:
+  // ),
+  // ProxyProvider<SettingsNotifier, BackupService>(
+  //   create: (_) => BackupService(),
+  //   update: (_, settings, backupService) => backupService!..update(settings),
+  // ),
   ProxyProvider<HttpService, AccountGeneratorService>(
     create: (_) => AccountGeneratorService(),
-    update: (_, httpService, accountGeneratorService) =>
-        accountGeneratorService!..update(httpService),
+    update: (_, httpService, accountGeneratorService) => accountGeneratorService!..update(httpService),
   ),
   ProxyProvider<AccountGeneratorService, CreateAccountBloc>(
     create: (_) => CreateAccountBloc(),
-    update: (_, accountGeneratorService, createAccountBloc) =>
-        createAccountBloc!..update(accountGeneratorService),
-  ),
```

Comment:
`AccountGeneratorService` and `CreateAccountBloc `

I think this is the logic of the signup, we would have to ask Fazard if he uses something from there or has already implemented it.